includes/class-wc-payments.php

```diff
@@ -457,6 +457,17 @@ public static function init() {
 			WC_Payments_Subscriptions::init( self::$api_client, self::$customer_service, self::get_gateway(), self::$account );
 		}
 
+		if ( WC_Payments_Features::is_express_checkout_enabled() ) {
```

Comment:
I see that the `WC_Payments_Features` class is so far used for feature flags (which are set as options). But this function sets up a precedent to use it as a "utility" class for settings that might be enabled/disabled.

Since there's only one usage of `WC_Payments_Features::is_express_checkout_enabled()`, what do you think about modifying this with inline checks?
i.e.: 
```suggestion
		$is_woopay_express_checkout_enabled = WC_Payments_Features::is_woopay_express_checkout_enabled();
		if ( $is_woopay_express_checkout_enabled || self::get_gateway()->get_option( 'payment_request' ) ) {
```

And below:

```
if ( $is_woopay_express_checkout_enabled ) {
```

Reply:
Thanks for pointing this out! I see what you mean and it does make more sense to handle these checks inline for now. Changed in https://github.com/Automattic/woocommerce-payments/pull/5520/commits/ba2e62020f30d1383f86f70a26f5d1a029b77368.